### PR TITLE
add tests for halveString

### PR DESCRIPTION
**Pull request by weirichs**

### CODE REVIEW

**Review by beckerbenj — CHANGES_REQUESTED** [review]
I like the tests, just some minor formalities 👍

File: tests/testthat/test_halveString.R [hunk 1]
```diff
@@ -0,0 +1,20 @@
+context("halveString")
```
[the remark this answers is not in the document]
Comment (beckerbenj) [reply tests/testthat/test_halveString.R:1]: see also [context](https://testthat.r-lib.org/reference/context.html)

[the remark this answers is not in the document]
Comment (weirichs) [reply tests/testthat/test_halveString.R:1]: Will be done (second option)

File: tests/testthat/test_halveString.R [hunk 2]
```diff
@@ -0,0 +1,20 @@
+context("halveString")
+
+str <- c("a__b__c", "a___m___k__")
+d1  <- halveString(str, pattern = "__", colnames= c("a", "b"))
```
[the remark this answers is not in the document]
Comment (weirichs) [reply tests/testthat/test_halveString.R:4]: Will be done, sir!

**Pull request merged**